tools/SourceKit/cmake/modules/AddSwiftSourceKit.cmake

```diff
@@ -367,6 +381,12 @@ macro(add_sourcekit_framework name)
                                DESTINATION lib${LLVM_LIBDIR_SUFFIX}
                                COMPONENT ${SOURCEKITFW_INSTALL_IN_COMPONENT}
                                USE_SOURCE_PERMISSIONS)
+    swift_is_installing_component("${SOURCEKITFW_INSTALL_IN_COMPONENT}" is_installing)
```

Comment:
Can we lift these out of the `if` since it's on both branches anyway?

Reply:
Sure.